### ISSUE TITLE
Scrollable content for iPhone 6/6 Plus

### ISSUE THREAD

**Issue by lg365** [mcasimir/mobile-angular-ui#158]
When adding the scrollable scrollable-content class, data can be scrolled on all devices apart from the iPhone 6 and the iPhone 6 Plus. This includes all other iOS devices, Safari on a desktop etc.

**Comment by ElleryW** [comment]
This works fine on my iPhone 6

**Comment by joshribakoff** [comment]
Scrolling for me worked on desktop Chrome, but scrolling did not work for me when running my page inside of a phonegap app. I removed `ui-prevent-touchmove-defaults` from my body tag, which was shown in the demo. This fixed scrolling for me on iphone 6 running my page inside a phonegap app.

**Comment by okku** [comment]
ok I see some really buggy behaviour for scrolling. I see it the app I am developing as well as in the demo app, its visible at least in iphone 6 plus and ipad device (in safari and chrome) as well as in simulator.  See video: https://youtu.be/161bFhj48hM

**Comment by workzentre** [comment]
Yeah same issue here... Scrolling is inconsistent... Exactly same behavior reported by okku... Anyone came to a solution to this? Thanks!

**Comment by okku** [comment]
Ok finally solve this!!!
Its caused by the gesture module, so if you don't need gestures just DON'T incude: mobile-angular-ui.gestures.js.
